cmd/zoekt-sourcegraph-indexserver/protos/sourcegraph/zoekt/configuration/v1/configuration.proto

```diff
@@ -136,8 +139,15 @@ message ListRequest {
 
 // ListResponse is the response from the List RPC.
 message ListResponse {
-  // repo_ids is the list of repository IDs that the client should index.
-  repeated int32 repo_ids = 1;
```

Comment:
This isn’t backward compatible, is that concerning? During a rollout that might mean zoekt can no longer get indexing info for a moment. 

If we decide to keep both we can remove it after the next release.

Reply:
I was thinking about that too, but came to the conclusion that this would only happen during rollout and not cause any errors. 

We could keep both and just set tenant to 1 if we receive messages of the old type, is that what you meant?

Reply:
yep for example! I'm not feeling super strongly about this, if we think this generally goes smooth enough, if this could cause repos to be trashed that might be sad :)

Reply:
Updated the code to take care of backward compatibly. I ran it against sg/sg main without multi-tenancy and it worked as expected.